src/frontend/src/flows/verifiableCredentials/index.ts

```diff
@@ -334,28 +365,28 @@ const getValidatedIssuerDerivationOrigin = async ({
     }
   | { kind: "ok"; origin: string }
 > => {
-  const result = await vcIssuer.getDerivationOrigin({ origin: issuerOrigin });
+  const derivationOriginResult = await vcIssuer.getDerivationOrigin({
```

Comment:
I took the liberty of renaming a few variables in this function because I think it makes following the logic a bit easier.